net/nimble/host/mesh/src/testing.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stddef.h>
+
+#include "mesh/testing.h"
+#include "mesh/glue.h"
+
+static sys_slist_t cb_slist;
+static sys_slist_t cb_slist = SYS_SLIST_STATIC_INIT(cb_slist);
```

Comment:
shouldnt it be

static sys_slist_t cb_slist = SYS_SLIST_STATIC_INIT(&cb_slist);